dashboard/app/views/pd/workshop_mailer/teacher_enrollment_receipt.html.haml

```diff
@@ -16,12 +16,14 @@
     curriculum.
 - elsif @workshop.ayw? || ([Pd::Workshop::COURSE_COUNSELOR, Pd::Workshop::COURSE_ADMIN].include? @workshop.course)
```

Comment:
This looks like it will impact AYW workshops too.

Reply:
Good point sorry about that! Fixing now